flow/envs/green_wave_env.py

```diff
@@ -9,7 +9,7 @@
 
 from gym.spaces.box import Box
 from gym.spaces.discrete import Discrete
-from gym.spaces.tuple_space import Tuple
+from gym.spaces.tuple import Tuple
```

Comment:
```suggestion
from gym.spaces.tuple_space import Tuple
```
I guess maybe you're running a different version of gym? Feel free to either push your version to the `requirements.txt` and `environment.yml` file or push the suggestion